common/data/packages_minimal.json

```diff
@@ -1,15 +1,15 @@
 {
-    "packages":["glibc", "zlib", "filesystem", "gmp", "mpfr", "libgcc", "libstdc++","libgomp", 
```

Comment:
These pacakges were not removed. A new json file will be generated including all the dependent packages of packages in the common/data/packages_minimal.json.